aci-preupgrade-validation-script.py

```diff
@@ -613,7 +618,10 @@ def icurl(apitype, query):
     logging.debug('response: ' + str(response))
     imdata = json.loads(response)['imdata']
     if imdata and "error" in imdata[0].keys():
```

Comment:
```suggestion
    if imdata and "error" in imdata[0]:
```

`"key" in dict` is encouraged especially in python2 which is unfortunately still used on older versions of APIC on which this script is going to run.